### ISSUE TITLE
CSS

### ISSUE THREAD

**Issue by thecodeduchess** [Smeallio/colmar#1]
Can you screenshot your website home page and post it here in a comment? 

I do not see an CSS styling when I open up your project in the broswer. I only see the text mainly. I'm not sure I could judge your visual work properly but I can make comments on your coding base. The linking looked fine but maybe you are missing bootstrap link or something?

**Comment by Smeallio** [comment]
![alex-colmar](https://user-images.githubusercontent.com/23176176/42580681-12fc4638-84f9-11e8-9fd0-e406eb202dd7.PNG)

**Comment by Smeallio** [comment]
Yeah, I've noticed this when I push code to GitHub. CSS and images work fine on my local drive but seem to disconnect when I push to GitHub. I can look into this a little more today.
